src/test/java/edu/iu/uits/mail/MockitoExtension.java

```diff
@@ -0,0 +1,64 @@
+package edu.iu.uits.mail;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolver;
+import org.junit.jupiter.api.extension.TestInstancePostProcessor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.lang.reflect.Parameter;
+
+import static org.mockito.Mockito.mock;
+
+public class MockitoExtension implements TestInstancePostProcessor, ParameterResolver {
```

Comment:
Can you provide some information on what this is used for?  It would be great to have some javadoc for it, especially if we need it due to some documentation in Mockito.

Reply:
Actually, this might be an outdated way of bringing in Mockito for JUnit 5.  You might check out https://solidsoft.wordpress.com/2018/03/27/convenient-mocking-in-mockito-with-junit-5-the-official-way/